bin.src/dummy.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
 
-echo "Dummy!"
+echo "A dummy script to likely be replaced later."
 
```

Comment:
What's the point of having this "dummy" script instead of just not having anything?

Reply:
Almost certainly can be removed.